addons/account/models/account_bank_statement.py

```diff
@@ -653,28 +661,46 @@ def auto_reconcile(self):
         self.ensure_one()
```

Comment:
match_recs = self.get_reconciliation_proposition()
if not len(match_recs) == 1:
    return False